spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -347,6 +347,29 @@ def main(argv=None):
             print_fail()
             print(err)
 
+    # Check version of FSLeyes
+    print_line('Check FSLeyes version')
+    cmd = 'fsleyes --version'
+    try:
+        status, output = run_proc(cmd, verbose=0, raise_exception=True)
+        # Fetch only version number (full output of 'fsleyes --version' is 'fsleyes/FSLeyes version 0.34.2')
+        fsleyes_version = output.split()[2]
+        print_ok(more=(" (%s)" % fsleyes_version))
+    # FSLeyes is not installed
+    except RuntimeError:
+        print("Not installed. While FSLeyes is not a requirement of SCT, we recommend to install it to easily "
+              "visualize processing outputs and/or to use SCT within FSLeyes. More info at: "
+              "https://spinalcordtoolbox.com/en/latest/user_section/fsleyes.html")
+    # Potentially sct_plugin.py related problem - see https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3426
+    except AttributeError as err:
+        print('AttributeError exception occurred. This exception might be related to sct_plugin.py script. More info '
+              'at: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3426')
+        print(err)
```

Comment:
Just a heads-up: `sct_plugin.py` only threw the error in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3426 when it was manually loaded using the "Run script" button in FSLeyes. So, I don't think there's any risk of this happening just by running `fsleyes --version`.

Also, the kinds of errors we might encounter in the future seem a bit hard to predict.... it was `AttributeError` this time, but in the future, it could be any number of exceptions. :sweat_smile: 

So, I would maybe say to get rid of this case, and catch everything else via the `except Exception as err` block.